Fusion filtering staggered PR4 (#39)
PediatricOpenTargets/OpenPBTA-analysis: analyses/fusion_filtering/04-project-specific-filtering.Rmd

    input: integer
  base_histology:
    label: "Base histology file"

>     value: data/histologies-base.tsv

Reviewer: ```suggestion
    value: data/histologies.tsv
```
should this be just histologies since we do not supply a base file in this repo at the moment?

Filtering for general cancer specific genes
Fusions with genes in either onco

> This notebook assumes you are in Openanalysis project folder structure.

Reviewer: ```suggestion
This notebook assumes you are in OpenPedCan-analysis project folder structure.
```

# Max number of times gene can be fused per sample
limitMultiFused<-params$limitMultiFused

> print("Raw calls from STARfusion and Arriba for OpenTarget")

Reviewer: ```suggestion
print("Raw calls from STARfusion and Arriba for OpenTargets")
```
